src/edu/wright/cs/raiderplanner/model/Person.java

```diff
@@ -140,6 +140,9 @@ public Person(String salutation, String givenNames, String famName,
 	public Person(String salutation, ArrayList<String> givenNames, String famName,
 			Boolean famNameLast, String newEmail) {
 
+		
```

Comment:
Is there a reason for these blank lines that I'm missing?

Reply:
I have no idea. My pull request didn't even merge with the original project so I have no clue.